src/audio/pcm_converter/pcm_converter_hifi3.c

```diff
@@ -678,7 +678,11 @@ static void pcm_convert_f_to_s16_lin(const void *psrc, void *pdst,
 		XT_xtfloat_loadip(x, in, sizeof(x));
 
```

Comment:
Firstly please trim the commit subject to around 100 chars. 

E.g: `pcm_converter: Fix architectures not supporting XT_ROUND_S`

Please also, fix checkpatch.pl issues. Also, please be patient.

Reply:
Some obvious problems, I will update later